Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM lukemathwalker/cargo-chef:latest-rust-1 AS chef
+WORKDIR /app
+
+FROM chef AS planner
+COPY . .
+RUN cargo chef prepare --recipe-path recipe.json
+
+FROM chef AS builder 
+RUN rustc --version; cargo --version; rustup --version
+
+# RUN curl -sLO https://github.com/tailwindlabs/tailwindcss/releases/latest/download/tailwindcss-linux-x64 && \
+#   chmod +x tailwindcss-linux-x64 && \
+#   mv tailwindcss-linux-x64 tailwindcss
```

Comment:
Commented out cause we are taking the easy approach of using Tailwind CSS from a CDN, but we will likely switch this eventually